### ISSUE TITLE
Error with Rails 5.1.4

### ISSUE THREAD

**Issue by dogweather** [jcamenisch/ENV_BANG#25]
```
env_bang-rails.rb:13:in `block in <class:Railtie>': private method `warn' called for nil:NilClass (NoMethodError)
```

I added the gem to the Gemfile, ran `bundle update`, and Rails won't boot up.

**Comment by jcamenisch** [comment]
Hey @dogweather, do you get the same symptoms with a fresh Rails app on that version?

**Comment by jcamenisch** [comment]
@dogweather, this issue is addressed in 0.4.2. You can also avoid that codepath by created a `config/env.rb` file.
